tests/src/tools/ReadyToRun.SuperIlc/ComputeManagedAssemblies.cs

```diff
@@ -21,12 +22,23 @@ public static IEnumerable<string> GetManagedAssembliesInFolder(string folder)
         }
     }
 
+    static ConcurrentDictionary<string, bool> _isManagedCache = new ConcurrentDictionary<string, bool>(StringComparer.OrdinalIgnoreCase);
+
     public static bool IsManaged(string file)
     {
         // Only files named *.dll and *.exe are considered as possible assemblies
         if (!Path.HasExtension(file) || (Path.GetExtension(file) != ".dll" && Path.GetExtension(file) != ".exe"))
             return false;
 
+        bool isManaged;
+        lock (_isManagedCache)
```

Comment:
Is the locking necessary for `ConcurrentDictionary`? This should all work without it.

Reply:
Good point, removed in 8th commit.